src/scielo/bin/xml/prodtools/xc.py

```diff
@@ -157,25 +157,36 @@ def convert_package(self, package_path):
 
         output_path = mkdtemp()
 
-        pkg = self._create_package_instance(source=xml_path, output=output_path)
-        pkg_name = pkg.package_folder.name
-
         try:
-            result = self.proc.convert_package(pkg)
-            scilista_items, xc_status, mail_info = result
-        except Exception as e:
+            package = self._create_package_instance(source=xml_path, output=output_path)
+            scilista_items, xc_status, mail_info = self.proc.convert_package(package)
+        except ScieloPackageError:
+            self.inform_failure(
+                package_path,
+                "Could not create package from source path '%s'." % package_path,
+            )
+            if self.config.queue_path is not None:
+                fs_utils.delete_file_or_folder(package_path)
+        except Exception:
+            self.inform_failure(
+                package_path, "Could not convert package '%s'." % package_path
+            )
             if self.config.queue_path is not None:
                 fs_utils.delete_file_or_folder(package_path)
-            self.inform_failure(pkg_name, e)
-            raise e
         else:
-            if len(scilista_items) > 0:
-                acron, issue_id = scilista_items[0].split(' ')
-                if xc_status in ['accepted', 'approved']:
-                    self._update_scilista(pkg_name, scilista_items)
-                    self._update_website_files(pkg_name, acron, issue_id)
-                self._mail_results(pkg_name, mail_info)
-                self._update_report_files(pkg_name, acron, issue_id)
+            if scilista_items is None or len(scilista_items) == 0:
+                logger.debug("Skping website update because the scilista is empty.")
```

Comment:
Trocar "Skping website update because the scilista is empty." por
"Skping continue because the scilista is empty."

O programa faz opcionalmente atualizar o site remoto e atualizar a scilista do gerapadrao QA.
Mas obrigatoriamente ele gera os relatórios.

Reply:
Mas tudo isso se e somente se a quantidade de itens dentro da scilista for maior que `0`, certo?

Não entendi a mensagem `Skping continue because the scilista is empty.`, será que não é melhor fazer algo como: `The scilista is empty. Skipping website update and report generation`?

Pelo que eu consegui entender do `if` anterior, estas etapas são realizadas apenas se `len(scilista_items) > 0`.

Reply:
@joffilyfe o que quis dizer que ele não só faz atualização do website.

Reply:
@robertatakenaka, atualizei a mensagem do log.